EXOSIMS/util/CheckScript.py

```diff
@@ -22,14 +26,14 @@ def __init__(self, scriptfile, outspec):
                 script = open(scriptfile).read()
                 self.specs_from_file = json.loads(script)
             except ValueError as err:
-                vprint("Error: %s: Input file `%s' improperly formatted."%(self._modtype,
+                print("Error: %s: Input file `%s' improperly formatted."%(self._modtype,
```

Comment:
unless dmity has said so otherwise somewhere, we use vprint (I believe it was because we needed it for python2 to python3 upgrade comptabatility

Reply:
Will put the vprint back in this file with the next push as well.